2.preprocess_features/scripts/1.norm_fs_pycytominer.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform data preprocessing with pycytominer on each compartment file
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+
+import pandas as pd
+from pycytominer import normalize, feature_select
+from pycytominer.cyto_utils import infer_cp_features
+
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+# operations to perform for feature selection
+feature_select_ops = [
+    "variance_threshold",
+    "correlation_threshold",
+    "blocklist",
+    "drop_na_columns"
+]
+
+# Output directory for processed profiles
+output_dir = pathlib.Path("./profiles/processed_profiles")
+output_dir.mkdir(parents=True, exist_ok=True)
+
+# Path to CytoTable converted profiles for each object
+converted_profiles_dir = pathlib.Path("./profiles/converted_profiles/")
```

Comment:
```suggestion
converted_profiles_dir = pathlib.Path("./profiles/converted_profiles/").resolve(strict=True)
```

Consider applying `resolve(strict=True)` if this file is expected to exist when running this.

Reply:
100% great point, I am adding it now!